solr/bin/install_solr_service.sh

```diff
@@ -46,7 +46,7 @@ print_usage() {
   echo "    -u     User to own the Solr files and run the Solr process as; defaults to solr"
   echo "             This script will create the specified user account if it does not exist."
   echo ""
-  echo "    -f     Upgrade Solr. Overwrite symlink and init script of previous installation."
+  echo "    -f     Force upgrade of Solr. Overwrite symlink and init script of previous installation."
```

Comment:
Maybe part of this is I don't want to touch the install service ;-).  Do we have another bug?  Because it appears that you tried to upgrade with a -f, it failed, and told you to use -f ????

Reply:
Yeah, I was thinking if it is easy enough to migrate the file to a Java class as well, but that's another topic. For now it's fine if we don't touch it and just be aware of its existence.

> Do we have another bug? Because it appears that you tried to upgrade with a -f, it failed, and told you to use -f ????

I'm not sure how it should behave in case of `-f` and if it actually stands for upgrade, force or both. It looks like the action taken is a simple upgrade, but the param should be something like `--upgrade`, instead of `-f`. In that case the message would make sense. Failing to upgrade would tell the user to force the action with `-f`/`--force`.

Reply:
i am also somewhat thinking that maybe we don't touch `install_solr_service.sh`..   it's only run once, it has no tests, and I'm somewhat leery of taking it on.   What if we finish all the flag conflicts in the bin/solr area, and then, in a seperate ticket, we go back and fix up `install_solr_service.sh` in a single JIRA?